### PR TITLE
feat(api): adding purchase track method to manually track purchases o…

### PR DESCRIPTION
**Pull request by milos1290**
…utside IAPs.

### CODE REVIEW

**Comment by milos1290** [comment]
Done.

File: AndroidSDK/src/com/leanplum/Leanplum.java [hunk 1]
```diff
@@ -1553,6 +1553,34 @@ public static void track(final String event, double value, String info,
...
+      }
+
+      final Map<String, String> requestArgs = new HashMap<>();
+      if (!TextUtils.isEmpty(currencyCode)) {
+        requestArgs.put(Constants.Params.IAP_CURRENCY_CODE, currencyCode);
+      }
+
+      LeanplumInternal.track(event, value, null, params, requestArgs);
+    } catch (Throwable t) {
+      Util.handleException(t);
```
Comment (Enuviel) [inline AndroidSDK/src/com/leanplum/Leanplum.java]: May be to put a Log?

Comment (benmarten) [reply AndroidSDK/src/com/leanplum/Leanplum.java]: Where?

Comment (Enuviel) [reply AndroidSDK/src/com/leanplum/Leanplum.java]: before handleException , something like Log.e("trackPurchase - Failed to track purchase event.");

**Pull request merged**